package.json

```diff
@@ -73,7 +73,7 @@
   },
```

Comment:
The package name will have to change for the NPM publish to work correctly from codeship

https://github.com/PropellerAero/cesium/blob/main/package.json#L2

Reply:
dang